packages/create-sitecore-jss/src/templates/angular-sxp/angular.json

```diff
@@ -0,0 +1,13 @@
+{
+  "projects": {
+    "<%- appName %>": {
+      "architect": {
+        "build": {
+          "options": {
+            "styles": ["node_modules/bootstrap/dist/css/bootstrap.min.css"]
```

Comment:
Do you know if this may overwrite the entirety of angular.json from base template?

Reply:
yep, tested and it works fine for both sxp and xmcloud addons